pages/principles-practices-idioms.md

```diff
@@ -64,6 +64,12 @@ The goal is to __automate at the lowest level possible, but no lower__. Shoot
 for an appropriate balance, not an ideal one. There will always be a need for
 manual testing; the point is to get the most out of it, not to eliminate it.
 
+Given rigorous test size definitions and appropriate labeling of tests of
```

Comment:
Perhaps "labeling or separation"? This can be as simple as having different test sizes in different directories (such as the natural separation between regular tests and selenium tests).